setup.py

```diff
@@ -46,7 +46,7 @@ def load_requirements(*requirements_paths):
     return list(requirements)
 
 
-VERSION = '1.3.8'
+VERSION = '2.0.0'
```

Comment:
Not sure if we really need to go to 2.0.0 since I don't believe this breaks any existing behavior.  Maybe 1.4.0 would be more appropriate?

Reply:
Sure, done @azarembok 

I considered a change in behavior big enough but I have no strong opinions on that.